src/app/card/card-old.ts

```diff
@@ -5,10 +5,5 @@
  */
 import {Component} from "@angular/core";
 
-@Component({
-    moduleId: module.id,
-    selector: "clr-card-old-demo",
-    styleUrls: ["./card.demo.css"],
-    templateUrl: "./card-old.html"
-})
+@Component({selector: "clr-card-old-demo", styleUrls: ["./card.demo.scss"], templateUrl: "./card-old.html"})
 export class CardOldDemo {}
```

Comment:
@hippee-lee: Good point. This is present because we did a non breaking change for cards. There are other such demos too. We need to clean them up as the deprecation period is already done for most of the old demos. But I think we should do that in a separate PR so that we can review whats needed and whats not.